src/exchanges/crdtExchange.ts

```diff
@@ -464,6 +458,64 @@ class CrdtManager {
     }
   }
 
+  private persistMutations() {
+    console.log('persistMutations', this.#mutations);
+    const mutationOps = Array.from(this.#mutations.values());
+
+    const persistedMutations = mutationOps.map((mut) => {
+      return {
+        query: print(mut.query),
+        variables: mut.variables,
+      }
+    });
+    this.#storage.writeMetadata!(persistedMutations);
```

Comment:
The storage engine we are using does not return a promise, but does write data async. We fire off a write but continue before we know that it was actually successful.

Should we address this now in this PR, or simply make a card for this to deal with later? It doesn't seem like this will end up being too much of an issue when we are using mmkv in our app.